medusa/subtitle_providers/legendastv.py

```diff
@@ -477,3 +477,12 @@ def download_subtitle(self, subtitle):
 
         # extract subtitle's content
         subtitle.content = fix_line_ending(subtitle.archive.content.read(subtitle.name))
+
+
+def raise_for_status(r):
+    # When site is under maintaince and http status code 200.
+    if 'Em breve estaremos de volta' in r.text or r.status_code == 503:
+        # For now raise Timeout until the upstream PR gets merged that catches ServiceUnavailable
+        raise Timeout
```

Comment:
The requested change is to fix the issue properly.

Don't expect always a solution from my side. I'm not working on this and I not in front of my ide with the system running to evaluate the possibilities. 
What I know is that a timeout is completely different to 503 or service unavailable. And all of this is just a nasty workaround because of the way we handle logging and user notifications.